comictalker/comiccacher.py

```diff
@@ -183,7 +183,7 @@ def get_search_results(self, source_name: str, search_term: str) -> list[ComicVo
                     count_of_issues=record[7],
                     start_year=record[8],
                     image_url=record[9],
-                    aliases=record[10].split("\n"),
+                    aliases=record[10].split("\n") if record[10] else [],
```

Comment:
Another suggestion for all of these: Use JSON array? If there has to be this messing around to get "clean" data out. It's already being used for credits.

Reply:
See my comment on the other thread discussing split